tests/cli/NewTestObject/DeltaEx.cs

```diff
@@ -73,13 +73,9 @@ public void FromData(Apache.Geode.Client.DataInput DataIn)
             FromDataCount++;
         }
 
-        public Int32 ClassId
+        public String Type
```

Comment:
Why define this on .NET? All .NET classes already have a `GetType()` method that we can use, similar to the Java side where we use `getClass()`. Remove this method from the interface.

Reply:
See previous comments above. 
```
        public String Type
        {
            get { return this.GetType().ToString(); }
        }
```